tdrs-frontend/src/reducers/reports.js

```diff
@@ -1,16 +1,24 @@
-import { SET_YEAR } from '../actions/reports'
+import { SET_SELECTED_STT, SET_SELECTED_YEAR } from '../actions/reports'
 
 const initialState = {
-  year: 2020,
+  year: '',
+  stt: '',
```

Comment:
To match the mockup, initial state of `year` should not be 2020 as nothing should be selected. Also these should be strings (instead of `null`) to avoid an error in the `inputs` these are passed to.